src/+otp/+brusselator/BrusselatorProblem.m

```diff
@@ -1,34 +1,55 @@
 classdef BrusselatorProblem < otp.Problem
-    % BrusselatorProblem A two-variable model for a system of autocatalytic
-    %   reaction equations
+    %BRUSSELATORPROBLEM A two-variable model for an autocatalytic reaction
+    %   The Brusselator chemical reaction is given by
     %
     %        A → X
     %    B + X → Y + D
     %   2X + Y → 3X
     %        X → E
     %
-    %   This system can be reduced to the following two equations
+    %   With the assumption that all reaction rates are one and the concentrations of A and B are constant parameters,
+    %   this system can be modeled by the following two differential equations.
     %
     %   X' = 1 - (b + 1)X + aX²Y
     %   Y' = bX - aX²Y
     %
-    %   where a and b are scalings such as to combine the concentrations of
-    %   the excess reactants A and B and the reaction rates.
+    %   Here, X and Y are concentrations of autocatylitic species of interest. Equations for species D and E are not
+    %   necessary as they can be deduced from X and Y.
+    %
+    %   Problem Properties:
+    %       Type: ODE
+    %       Number of Variables: 2
+    %       Stiff: no
     %
     %   Bibliography:
-    %   Ault, Shaun, and Erik Holmgreen. "Dynamics of the Brusselator."
-    %   Math 715 Projects (Autumn 2002) (2003): 2.
+    %   Lefever, R., and G. Nicolis. "Chemical Instabilities and Sustained Oscillations." Journal of Theoretical
```

Comment:
Do we have a preferred citation style, or anything goes?

Reply:
MLA
https://github.com/ComputationalScienceLaboratory/ODE-Test-Problems/blob/master/docs/CONTRIBUTING.md#documentation

Reply:
Oh perfect, I was going to suggest adding it to Contributing.md, but looks like you have your bases covered, fantastic!